acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+    def __init__(self, base_url: str, timeout: float = 30.0) -> None:
+        """Initialize the client with the base URL of the index service
+
+        Args:
+            base_url: Base URL of the acoustid-index service
+            timeout: Default timeout for HTTP requests in seconds
+        """
+        self.base_url = base_url.rstrip("/") + "/"
+        self.timeout = timeout
+        self.session = aiohttp.ClientSession()
+
+    async def __aenter__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__aenter__`

(ANN204)